### PR TITLE
My local branch

### PR DESCRIPTION
**Pull request by whyte2**
Removed: requirements.txt

Updated: app.py, now contains logic for how to handle events that contain POST and GET httpMethod requests. 

Updated: template.yaml, now has a new dynamoDB table called "myFoodTable", the single lambda function now has two trigger events, GET and POST. LambdaRole also added.

Added: Pipfile for dependencies. 

Testing: locally with docker image and in postman.

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -5,37 +5,77 @@ Description: >
...
+                  Action:
+                    - cloudwatch:*
+                    - s3:*
+                    - logs:*
+                    - ssm:*
+                    - dynamodb:*
+                    - cognito-idp:*
+                    - es:*
+                    - ec2:*
+                    - lambda:InvokeFunction
```
Comment (ryohang) [inline template.yaml]: in any client project, we can't use * as part of policy, it has to be specific. All you need to do is to understand your lambda function requirement.

**Review by ryohang — APPROVED** [review]

File: hello_world/app.py [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
+        item = {"FoodID": FoodId,
+                "county": country,
+                "season": season,
+                "name": name}
+
+        table.put_item(Item=item)
+        
+        return {
+            "statusCode": 200,
+            "body": json.dumps({"message": "added to db"})
```
Comment (ryohang) [inline hello_world/app.py]: you need to return object. We need to discuss this part. @whyte2 make sure you remind me today.

**Comment by whyte2** [comment]
Hi Ryo, Pr sent.

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+boto3 = "*"
+requests = "*"
+botocore = "*"
+dynamodb-json = "*"
```
Comment (ryohang) [inline Pipfile]: lock the library to specific version all the time.

**Review by ryohang — CHANGES_REQUESTED** [review]

**Comment by whyte2** [comment]
New pr added.

**Review by ryohang — APPROVED** [review]

**Pull request merged**